cli/src/main/java/com/liferay/blade/cli/BladeCLI.java

```diff
@@ -336,6 +346,44 @@ public void trace(String s, Object... args) {
 		}
 	}
 
+	public boolean updateAvailable() throws IOException {
+		boolean available = false;
+
+		VersionCommand versionCommand = new VersionCommand(this);
+
+		String bladeCLIVersion = versionCommand.getBladeCLIVersion();
+
+		out("The current version of blade is " + bladeCLIVersion + System.lineSeparator());
+
+		boolean fromSnapshots = false;
+
+		if (bladeCLIVersion == null) {
+			bladeCLIVersion = "0.0.0";
+		}
+
+		fromSnapshots = bladeCLIVersion.contains("SNAPSHOT");
+
+		String updateVersion = "";
+
+		try {
+			updateVersion = UpdateCommand.getUpdateVersion(fromSnapshots);
+
+			available = UpdateCommand.shouldUpdate(bladeCLIVersion, updateVersion);
+
+			if (available) {
+				out(
+					"Run \'blade update" + (fromSnapshots ? " --snapshots" : "") + "\' to start using " +
+						(fromSnapshots ? "the latest snapshot " : " ") + "version " + updateVersion +
+							System.lineSeparator());
+			}
+		}
+		catch (IOException ioe) {
+			available = false;
+		}
```

Comment:
Should we pass this error to `getBladeCLI().error(ioe)`?

Reply:
This would cause an AssertionError in the HelpCommandTest.